diagnostic.js

```diff
@@ -29,26 +29,82 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const db = mongoose.connection;
+const done = function() {
+  db.close();
+};
```

Comment:
`db`  and `done` are not necessary here - they were part of the UI functionality of the command line app. We needed to ensure that the Mongo connection was closed each time the script finished running so that there weren't lots of open connections at the end of the day. In this case, closing the connection after doing a single CRUD action would prevent our app from doing anything else.